keps/NNNN-kep-template/README.md

```diff
@@ -156,6 +156,13 @@ should help to ensure that the tone and content of the `Summary` section is
 useful for a wide audience.
 
 A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the
+[kubernetes/community documentation style guide]. In particular, wrap lines
```

Comment:
```suggestion
[documentation style guide]. In particular, wrap lines
```

Reply:
done, though FTR note that that document currently explicitly identifies itself as being "for content in the Kubernetes github [community repository](https://git.k8s.io/community)."

Reply:
@danwinship -- Fair point. We should probably reconcile documentation.